api/src/pcapi/utils/requests.py

```diff
@@ -100,4 +100,4 @@ def __init__(self, *args, **kwargs):  # type: ignore [no-untyped-def]
         unsafe_adapter = HTTPAdapter(max_retries=unsafe_retry_strategy)
         self.mount("https://www.demarches-simplifiees.fr", safe_adapter)
         self.mount(settings.UBBLE_API_URL, safe_adapter)
-        self.mount("https://api.batch.com", unsafe_adapter)
+        self.mount(settings.BATCH_API_URL, unsafe_adapter)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
:partying_face: